test/src/unittests/tonal/test_pitchsaliencefunctionpeaks.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env python
+
+# Copyright (C) 2006-2021  Music Technology Group - Universitat Pompeu Fabra
+#
+# This file is part of Essentia
+#
+# Essentia is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Affero General Public License as published by the Free
+# Software Foundation (FSF), either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the Affero GNU General Public License
+# version 3 along with this program. If not, see http://www.gnu.org/licenses/
+
+
+from numpy import *
+from essentia_test import *
+
+class TestPitchSalienceFunctionPeaks(TestCase):
+  
+    def testInvalidParam(self):
+        self.assertConfigureFails(PitchSalienceFunctionPeaks(), {'binResolution': -1})
+        self.assertConfigureFails(PitchSalienceFunctionPeaks(), {'maxFrequency': -1})
+        self.assertConfigureFails(PitchSalienceFunctionPeaks(), {'minFrequency': -1})
+        self.assertConfigureFails(PitchSalienceFunctionPeaks(), {'referenceFrequency': -1})                
+
+    def testEmpty(self): 
+        self.assertRaises(RuntimeError, lambda: PitchSalienceFunctionPeaks()([]))
+    
+    # Check for empty output arrays for a salience function with all zero values
+    def testZeros(self): 
+        bins, values = PitchSalienceFunctionPeaks()(zeros(128))
+        self.assertEqualVector(bins, [])
+        self.assertEqualVector(values, [])    
```

Comment:
PitchSalienceFuncionPeaks is just a wrapper for PeakDetection, so these tests should be already covered by PeakDetection itself (we can check) and are redundant here.